tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "stripInternal": true,
```

Comment:
We aren't generating declaration files yet, so I think we can remove this too(?)

Reply:
:heavy_check_mark: